ui/app/machines/secrets-machine.js

```diff
@@ -43,7 +43,7 @@ export default {
             cond: type => ['pki', 'aws', 'ssh'].includes(type),
           },
           secret: {
-            cond: type => ['cubbyhole', 'database', 'gcp', 'kv', 'nomad', 'rabbitmq', 'totp'].includes(type),
+            cond: type => ['kv'].includes(type),
```

Comment:
is kv-v2 a separate type here? We probably also want the legacy `generic` type (it's what kv used to be called).

Reply:
For now I just have what's reachable in the UI and it sends `kv`. In the future we'll definitely include the others since we'll have to discern between them.